src/main/java/org/cascadebot/cascadebot/messaging/MessagingUI.java

```diff
@@ -196,7 +199,7 @@ public void sendTracksFound(List<AudioTrack> tracks) {
         }
     }
 
-    public void checkPlaylistOrSong(String input, List<AudioTrack> tracks, CommandContext context) {
+    public void checkPlaylistOrSong(String input, List<AudioTrack> tracks, CommandContext context, boolean top) {
```

Comment:
It isn't quite clear what "top" means in this context. Perhaps either add a JavaDoc or name it something a little more descriptive?